### DISCUSSION TITLE
Is SAML natively integrated with Janssen ?

### DISCUSSION THREAD

**Discussion by brunoflament** [JanssenProject/jans#5139] [Q&A] [answered]
Hello,

Is SAML natively integrated with Janssen ?
Will it be soon?

Cordially
Bruno F.

**Comment by nynymike** [comment, marked as answer]
We are working on adding KeyCloak as a component of Janssen.  Similar to Gluu 4, you'll be able to use the Jans Config API to add "SAML Relationships". For now, if you want SAML, I would use KeyCloak directly.

**Reply by brunoflament** [reply]
Hello, ok thanks
